vscode-dotnet-runtime-library/src/EventStream/OutputChannelObserver.ts

```diff
@@ -127,6 +127,23 @@ export class OutputChannelObserver implements IEventStreamObserver {
         // Nothing to dispose
     }
 
+    private updateDownloadIndicators(installKey : string | null | undefined)
+    {
+        if(installKey && installKey !== 'null')
```

Comment:
Is it possible that installKey could be empty?
```suggestion
        if(installKey && installKey !== 'null' && installKey !== '')
```

Reply:
'' has a 'truthiness' value of 'false', so this additional condition is unnecessary, but this is good thinking because it could be '' :)